source/includes/fundamentals/code-examples/connection/OData.cs

```diff
@@ -0,0 +1,65 @@
+// start-controller
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.OData.Routing.Controllers;
+using MongoDB.AspNetCore.OData;
+using MongoDB.Driver;
+using ODataTest.Models;
+
+namespace ODataTest.Controllers;
+
+public class RestaurantsController : ODataController
+{
+    private readonly IQueryable<Restaurant> _restaurants;
+
+    public RestaurantsController(IMongoClient client)
+    {
+        var database = client.GetDatabase("sample_restaurants");
+        _restaurants = database.GetCollection<Restaurant>("restaurants")
+            .AsQueryable();
+    }
+
+    // Register Get endpoint and set max documents to 5
```

Comment:
[s] I think these narrative comments can be updated to align with the [Code Comment Style Guide](https://docs.google.com/document/d/1Bnhh1OsGRUuXo_6yVUASGWiSX6sWuqrEFxPP6sLZDHM/edit?tab=t.0#heading=h.xv6glj2lnzjh)
```suggestion
    // Registers Get endpoint and sets max documents to 5
```